easybuild/easyconfigs/a/avrocpp/avrocpp-1.11.1-GCC-11.2.0.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'Binary'
```

Comment:
If I look at the install_cmd, this seems to be a CMakeMake installation. Wouldn't it be possible to use the `CMakeMake` easyblock? Binary is really for binary installations, i.e. binaries that are not compiled, but only installed. That means (among other things) that e.g. rpath-checking is skipped.

Note that the CMakeMake easyblock supports specifying a generator (see [here](https://github.com/easybuilders/easybuild-easyblocks/blob/fce1775ad011a9ccc3ad6963cbd6f350af6c3301/easybuild/easyblocks/generic/cmakemake.py#L198)).

I think the only thing custom to this install_cmd is the change in directory. I guess you can do this using `preconfigopts=cd lang/c++ &&` or something similar. I'm not 100% sure what the default startddir for EasyBuild will be when building, but you may need to specify a similar `prebuildopts` and `preinstallopts`.